.gitignore

```diff
@@ -0,0 +1 @@
+.DS_store
```

Comment:
need to add `.idea` folder to `.gitignore` and delete the folder from the repo (currently it is in and it has 4 files in it - they all have to be removed from the repo)